Library/Homebrew/tab.rb

```diff
@@ -47,6 +47,12 @@ def self.create(formula, compiler, stdlib)
           "version_scheme" => formula.version_scheme,
         },
       },
+      "built_on"                => {
+        "os"         => OS.family,
+        "os_version" => OS.version&.to_s,
```

Comment:
Sounds like on Linux this won't be valuable but `glibc` and `gcc` may be.

Reply:
Yeah I will definitely include `glibc` and `gcc`. I just did `OS::Linux.os_version` because I thought "it was better than nothing" but I can easily make it a MacOS specific thing too.